examples/dao/scripts/deploy.js

```diff
@@ -29,8 +30,8 @@ async function run(runtimeEnv, deployer) {
 		`int:${maxDuration}`,
 		`str:${url}`,
 		`str:${daoName}`,
+		`int:${govTokenId}`,
```

Comment:
I think at here the correct value is `govToken.assetIndex`

Reply:
you are right